[Docs] External Network (#2671)
liqotech/liqo: docs/advanced/use-only-offloading.md

All shadow pods will have an **empty IP address**, and will not be selected as targets by any Kubernetes service.

(AdvancedUseOnlyOffloadingSpecificCluster)=

## Disable Networking for a specific Cluster

It is also possible to disable the networking module only for the peering with a specific cluster.
This can be done by adding the `--disable-networking` flag to the `liqoctl peer` command when establishing a peering.

```bash
liqoctl --kubeconfig=$CONSUMER_KUBECONFIG_PATH peer \
    --remote-kubeconfig $PROVIDER_KUBECONFIG_PATH \
    --disable-networking
```

The `--disable-networking` flag will prevent Liqo from configuring the networking between the clusters for that specific peering. While the resource reflection and offloading is still possible.

Alternatively, if you are using the [Manual Peering](/advanced/manual-peering) method, you can disable the networking module simply by skipping the Inter-cluster Network Connectivity step.

In that scenario, the Pod IPs will still be reflected in the remote cluster, but they will not be remapped. You can still connect to the remote pods using an external network tool.

> In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file at installation time or by adding a custom `VkOptionTemplate` resource to your consumer cluster.

Reviewer: ```suggestion
In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file at installation time or by adding a custom `VkOptionsTemplate` resource to your consumer cluster.
```

When this flag is set to `false`, the **IP addresses** of the remote pods **are not reflected** and both local and remote <a href="https://kubernetes.io/docs/concepts/services-networking/" target="_blank">EndpointSlices</a> resources are not populated.

All shadow pods will have an **empty IP address**, and will not be selected as targets by any Kubernetes service.

## Disable Networking for a specific Cluster

It is also possible to disable the networking module for a specific cluster.
This can be done by adding the `--disable-networking` flag to the `liqoctl peer` command when establishing a peering.

```bash
liqoctl --context=consumer peer --remote-kubeconfig <kubeconfig-provider> --disable-networking
```

> The `--disable-networking` flag will prevent the Liqo network CRDs from being created in the clusters, and the networking module will not be enabled for that specific peering. While the ResourceSlices and VirtualNodes will still be created.

Reviewer 2: I wonder whether it is better being less specific, saying something like:
```suggestion
The `--disable-networking` flag will prevent Liqo from configuring the networking between the clusters for that specific peering. While the resource reflection and offloading is still possible.
```

Alternatively, if you are using the [Manual Peering](/advanced/manual-peering) method, you can disable the networking module simply by skipping the Inter-cluster Network Connectivity step.

In that scenario, the Pod IPs will still be reflected in the remote cluster, but they will not be remapped. You can still connect to the remote pods using an external network tool.

> In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file or by adding a custom `VkOptionTemplate` resource to your consumer cluster.

Reviewer 2: ```suggestion
In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file at installation time or by adding a custom `VkOptionTemplate` resource to your consumer cluster.
```

> In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file or by adding a custom `VkOptionTemplate` resource to your consumer cluster.

Reviewer 2: I would also add an example of the commands to run to achieve this result.
